server/athenian/api/openapi/openapi.yaml

```diff
@@ -4039,3 +4102,118 @@ components:
           description: New name of the token.
           type: string
       type: object
+    TimeZone:
+      description: Local time zone offset in minutes, used to adjust `date_from`
+        and `date_to`.
+      minimum: -720
+      maximum: 720
+      type: integer
+    TimeDuration:
+      description: Time duration value.
+      pattern: ^\d+s$
+      type: string
+    PullRequestHistogramsRequest:
+      additionalProperties: false
+      description: Request of `/histograms/prs`.
+      properties:
+        for:
+          description: |-
+            Sets of developers and repositories for which to calculate the histograms.
+            The aggregation is `AND` between repositories and developers.
+            The aggregation is `OR` inside both repositories and developers.
+          items:
+            $ref: '#/components/schemas/ForSet'
+          type: array
+        metrics:
+          description: List of desired histogram topics.
+          items:
+            $ref: '#/components/schemas/PullRequestHistogramTopic'
+          type: array
+        scale:
+          $ref: '#/components/schemas/HistogramScale'
+        bins:
+          description: Number of bars in the histogram. 0 means automatic.
```

Comment:
what should we expect about the automatic mode? a default number for all the histograms? different value for each metric? does it depend on the data being processed?

wouldn't it be a good idea to document the automatic mode?

(not complaining, just trying to know what to expect if using this)

Reply:
There are several algorithms for determining the best number of histogram bins, e.g. Freedman-Diaconis. I have not started to write the code yet so I cannot detail here.

Reply:
Great! So just please document what could we expect when that part is ready.